### PR TITLE
Add integration test for lldb in the toolchain

### PR DESCRIPTION
**Pull request by michael-weng**
Test the following contracts with lldb:
1. Assert that getLldbProcess returns data structure of right fields
2. Assert that getLLDBLibPath returns the right library paths for various platforms

Issue: #1023

### CODE REVIEW

**Review by award999 — CHANGES_REQUESTED** [review]
The test scenarios look good, some minor style comments

File: test/integration-tests/debugger/lldb.test.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { expect } from "chai";
+import { getLLDBLibPath, getLldbProcess } from "../../../src/debugger/lldb";
+import { SwiftToolchain } from "../../../src/toolchain/toolchain";
+import { WorkspaceContext } from "../../../src/WorkspaceContext";
+import { SwiftOutputChannel } from "../../../src/ui/SwiftOutputChannel";
+
+suite("lldb contract test suite", () => {
+    test("getLldbProcess Contract Test, make sure the command returns", async () => {
+        const toolchain = await SwiftToolchain.create();
+        const workspaceContext = await WorkspaceContext.create(
```
[the remark this answers is not in the document]
Comment (michael-weng) [reply test/integration-tests/debugger/lldb.test.ts:24]: Done.

File: test/integration-tests/debugger/lldb.test.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            // If result is an array, assert that each element has a pid and label
+            result.forEach(item => {
+                expect(item).to.have.property("pid").that.is.a("number");
+                expect(item).to.have.property("label").that.is.a("string");
+            });
+        }
+    });
+
+    test("getLLDBLibPath Contract Test, make sure we can find lib LLDB", async () => {
+        const toolchain = await SwiftToolchain.create();
```
[the remark this answers is not in the document]
Comment (michael-weng) [reply test/integration-tests/debugger/lldb.test.ts]: Done.

**Review by award999 — APPROVED** [review]

**Comment by award999** [comment]
> Issue: #1023

Before we merge can you add more details of what was tested?

**Review by award999 — CHANGES_REQUESTED** [review]
@michael-weng can you rebase and run "npm test" locally again? BTW great example here of test with explicit setup (acquire toolchain), 1 action (get path), and a good isolated assertion(s)

**Comment by michael-weng** [comment]
> @michael-weng can you rebase and run "npm test" locally again? BTW great example here of test with explicit setup (acquire toolchain), 1 action (get path), and a good isolated assertion(s)

Done tests passed.

**Pull request merged**